### DISCUSSION TITLE
Home Assistant Climate Code for EHYHBX08AAV3

### DISCUSSION THREAD

**Discussion by stokkie90** [Arnold-n/P1P2MQTT#70] [Q&A]
@Arnold-n I noticed you have exactly the same unit as I have. Do you by any chance have Climate control enabled in Home assistant and would you be able to share code. 

Might be an idea to introduce a folder in the repo with example Home assistant code for the different systems. 

I've already tried: https://github.com/Arnold-n/P1P2Serial/discussions/59 but it's not the same unit etc. As Daikin EHYHBX08AAV3 support cooling.

Thanks for the great work.

**Comment by Arnold-n** [comment]
You're welcome and thanks for your feedback! The solution in #59 is in my view a temporary solution until the ESP code auto-creates the control entity in HA - I am working on that code.

The code in #59  can be extended to cooling: "E36 6 xxx" is for setting LWT temperature in heating mode, and "E36 7 xxx" is for setting LWT temperature in cooling mode. I think "E35 4E 0" sets the system to heating and "E35 4E 1" sets the system to cooling, but currently I do not use cooling myself.

**Comment by Arnold-n** [comment]
With v0.9.46, climate control is now automatically set up in HA via MQTT config messages.
